src/append/console.rs

```diff
@@ -170,11 +185,18 @@ impl ConsoleAppenderBuilder {
             },
         };
 
+        let do_write = if let Writer::Tty(_) = writer {
+            true
+        } else {
+            ! self.tty_only
+        };
```

Comment:
NIT: This might read a little more fluently.
```rust
let do_write = writer.is_tty() || !self.tty_only;
...
impl Writer {
  fn is_tty(&self) -> bool {
    matches!(self, Self::Tty(_))
  }
}
```